### PR TITLE
Framework updates, improved Carthage support

### PR DESCRIPTION
**Pull request by runmad**
#### What this does

Removed static scheme, updated targets and schemes for improved Carthage support. 
I have also updated the README to reflect the changes.

@davidmuzi please review :eyes:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,17 +98,17 @@ The workspace includes the Mobile Buy SDK project.
 
 The Mobile Buy SDK includes a number of targets and schemes:
 
-* **Buy**: This is the Mobile Buy SDK static framework. This build is based on the current build configuration. To build a universal framework that can run on a device and on the Simulator and to be included in your app, please refer to the `Universal Framework` target below
+* **Buy**: This is the Mobile Buy SDK dynamic framework. Please refer to the installation section above
 
-* **Buy Dynamic**: This is the Mobile Buy SDK dynamic framework. Please refer to the installation section above.
+* **Buy Static** (target only): This is the Mobile Buy SDK static framework. This build is based on the current build configuration. To build a universal framework that can run on a device and on the Simulator and to be included in your app, please refer to the `Universal Framework` target below
```
Comment (davidmuzi) [inline README.md]: `Static Universal Framework` target below

**Comment by davidmuzi** [comment]
Tested for functionality - looks good.

**Comment by cocoahero** [comment]
:-1: 

```
Code Sign error: No code signing identities found: No valid signing identities (i.e. certificate and private key pair) matching the team ID “A7XGC83MZE” were found.
A shell task failed with exit code 65:
** BUILD FAILED **
```

Please do not include your own team ID...

**Comment by runmad** [comment]
Thanks, it'll be fixed for 1.2.1.

**Pull request merged**